radiotherapyai/__main__.py

```diff
@@ -12,10 +12,16 @@
 # implied. See the License for the specific language governing
 # permissions and limitations under the License.
 
+"""CLI entrance
 
-def app():
-    from . import _build
+Currently only utilised to build the docs
+"""
+
+from . import _build
 
+
+def app():
+    """The function called by the CLI commands `rtai` and `radiotherapyai`"""
     _build.build()
```

Comment:
If one looks at this from a Command pattern perspective, build() is a reasonable name as an alternative to "execute()", but _build would more typically have a name of _documentation, so that you end up with an invocation of
_documentation.build()

Or... one might have a name of _documentation_build with a function named execute, with an invocation of
_documentation_build.execute()
and that way you could have a variety of different commands that you would execute (or a master build that would then invoke a bunch of different command "classes" and tell them to "build()".

Nothing that has to change right away, if ever, but...

Reply:
Yup :slightly_smiling_face:, easy fix. Thanks @sjswerdloff :slightly_smiling_face:.

Fixed in 3563b63